app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -110,6 +110,22 @@ def import_package_profile_in_bulk(simple_packages)
         found
       end
 
+      def import_enabled_repositories(repos)
+        paths = repos.map do |repo|
+          if !repo['baseurl'].blank?
+            URI(repo['baseurl'].first).path
+          else
+            logger.warn("System #{@host.name} (#{@host.id}) attempted to bind to unspecific repo (#{repo}).")
+            nil
+          end
+        end
+        content_facet.update_repositories_by_paths(paths.compact)
+      end
+
+      def import_module_streams(module_streams)
+        # TODO: Implement this when module stream support is added
```

Comment:
Do we need this TODO and blank method?

Reply:
I can add it in https://github.com/Katello/katello/pull/7788 if you prefer. Just left it there as a place holder.